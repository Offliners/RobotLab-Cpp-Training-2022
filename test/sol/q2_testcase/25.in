603
15O
DIscoVERY
PLATO
endurAncE
GRAcE
supeRiOr
sometIme
WEAltHY
iNgenUity
vASt
ADvisORY
ToURIST
furnituRe
ObSERVE
MaSsACRE
PREFERABLE
iNTeND
SpIT
last
tighT
quarRel
circuS
continUe
PartIculAR
ChoruS
sERmON
bay
knifE
pACKAgE
DeScENdANT
paRalyze
Hom
BAGGAgE
hYMn
MeMORABLe
sTORy
SupERsTiTioN
aMeNNa
APOLogY
fiShErman
WHOM
ConFlicT
dairy
discard
PROFITAble
DistInCT
CONFIdE
10V6958
iMmUNE
Odds
sport
oil
pRESIdeNTial
capiTOl
twelve
iniTially
aSpECt
dazzle
hot
HANDY
RELy
aLBum
Beneath
LI5
GENDEr
COoPERAtIvE
sneaK
ax
eNVIous
REluCTanT
HEaL
PARACHuTe
banquet
nURsE
Enemy
Climax
gUesT
T05l
SOlar
distrust
approvE
n080
SpOT.
lAndlORD
COFFEE
tire
jeWElRY
RESORT
actIvity
BEgGaR
chaNcElLOr
CReaTIvITY
muSCulAR
certIficaTe
anonymous
unexpectEdly
TeACh
SLIpPER
sOcial
pReSeRvAtIOn
POSsIBiliTY
FAST
bLUNDER
DENOMiNatIOn
1Ew
aTtainment
AppAl
ViCioUs
then
crowD
rETURN
tHirst
trOop
web
accumuLATIon
paYDAy
sleepy
CUBE
cOnSuLT
TEXT
WhO
handbook
facTor
mOB
GROAN
AC0ui43
powDER
aIm
PennY
PRoTEcTive
TOAST
SHO5TEN
lEAKAGE
crosSroAd
Tempt
s0ElL
pOST-WAr
Costly
ANNUAL
TIMBeR
adAptaTion
reluctantly
aLive
BAr
5aj7mas
counselinG
either
DECK
bamboo
shOck
CONTEXt
0nflu22z4
pETItiON
PAssport
CRoUCH
ovErlOOK
REPEL
moIsTuRe
FIeA
pReScripTION
Voice
rEOrGANIzATiON
STOp
ReSPECTaBlE
Sq1156
toWEr
CARRoT
RECIpIENT
3L877
TelEScOpE
STream
PRACTITIONEr
2a19
shorten
ANO6y1OUS
BIoLOgIcAL
SYMPHONY
pLUMBeR
MaNdaRIn
honor
EXAMINE
SUIT
daddy/dad
waterfALl
fLOOR
frighteN
gifT
another
o65Up8T080
outlEt
coMMoNPLACE
scientist
BuRDEN
eARTH
lAment
SHOULD
ELectRician
heaLth
electRonics
BUFFeT
accOUntAnt
eAsTEr
deFinE
Dip
ComprEhEnsIon
team
41AcK9OW8
PR08ote
INTUITION
ALLERGY
breast
LIMITAtION
riGoRoUs
DECLINE
coNFEdeRaTIoN
0are7Ess
MANUFACTURE
70r55
GRAb
asS
ScaRE
bUTTON
Pig
PORTION
vidEO
waIt
ButtErFlY
whoLESaLE
bed
declaraTIOn
OvEREAT
PROVerb
warNing
brUtAl
aTtACk
COmPeTENcE
PRAcTICal
MARaTHoN
Energy
subseqUently
cook
captIvity
cREdIBLE
Astonish
buCk/DolLaR
STINGY
CHRIsTIANITy
HISTORIAN
A98315
DIE
mEntally
MiLD
asCErtAiN
ArGUE
HOOK
9A9d
DISCLOSURE
Goods
SOMEhOW
BEiJIng
sEason
pRiSON
STeEl
delINquent
SILLY
coNVEnienT
ORDINANCE
pave
jUNCTIOn
LeNGtHY
PeRsiSt
cousin
MANufAcTURer
hYGiENe
ambulaNce
graVe
coexist
antibodY
DEaf
22e497O3e
Card
WeEKDAY
trafFic
CaSH
p9091
haPPEn
StrIDE
DIFFERENTIAL
FaME
WaYside
pEAnut
architect
ESSENTIALLY
MotivAtIoN
great
sOrry
structure
mERRY
INDOORS
sANdWICH
MEDITaTIon
cOMmUNIcation
NOMINATION
bl5n4
InSTANtLY
2I612
CRAWL
CLaIm
hyPocrite
cOMmeNtAToR
CerEmOny
SupErB
asylum
3PARROW
AuDIence
ScrEam
MoDEL
SiNG
kILL
PROCESS
bAttlE
7as34G5
mONUMENT
SHady
tErrifiEd
FIELd
sAMPLE
sTEEr
knoT
prIvacy
NiNCOMPOoP
BIOGRAPHY
sUccESs
SPERM
bREACH
gIant
ExPlaIn
Honey
BlAnKEt
nuisaNCe
pavement
PhILoSOPHiCAl
c265
lantErN
hATCH
cHEF
glOBaL
buSY
ECOLOGy
corn
JEWEL
qUANTITY
TURN
SURVEY
SURFACE
COPe
DeVELOP
IDEOLoGy
Loan
PLUS
county
EgG
aftErward
wAllet
cl6n8c
937u904
NOMINeE
THESE
tOKEn
trick
rainfall
soUnD
cirCulaTION
iMITaTION
PoLl
fiu
ANALOGY
STARTLE
CONSENT
STUPID
RoMANce
PrONe
TWIST
smOg
Party
MiRrOr
HostILity
NOTORIOUS
O87422
SkatE
self
recognition
GUARDIaN
paSsIVE
remind
APPEAr
soLVE
ACREAgE
OUtSeT
progressIve
SUpERVISE
AngLE
pRosperous
DIVERSIOn
ConTENT
692y
coP
UNTOUCHED
SIGHT
Future
FEEL
adDitIoNaL
rOLl
FORTH
MAP
CONTACT
bend
GRAdUAL
shORtcoming
overcoAT
JOy
credibility
oRIGINALITY
travEl
apPRovAL
PICTURE
LitTLE
DiScoUrAGINg
generOsity
GIRAFfE
HUrRIcaNE
rIVER
HURL
list
DAMAGe
INTERPRET
domestic
CALCULATOr
MoMenT
PEril
avErage
BoDY
PASSION
R3056772
SHEEP
DEteRmInAtION
manNer
MaJoritY
StrONG
taIl
MaRch
cORREcT
complete
lease
donation
walK
POPULATE
account
hUsBand
CAUTIOUSLY
FamilY
PROnOUN
publication
slogan
stEwarD
oFFiCeR
SIniSTER
caLF
REfIneMeNT
BATh
downtown
department
AGONY
APpOINTMENT
DOWNWARDS
REScUe
amBUsh
refRain
CorrupTioN
WAItrEsS
express
MARVELOUs
dIAgNoSE
teRritoRy
LOV6
DIVoRCE
apparent
PrAIse
bOtHer
lunchEon
beef
opTiON
FORTUNE
CompARTMENT
lotion
geNuiNE
V94A7CY
PURPLE
BALcOnY
instEad
mUrdErEr
GRavEyARd
ATTENDANT
Bear
CONCENTRATE
BeT
unLIkE
PapA
adapt
PreJuDiCE
insPiRinG
COnTIneNt
outrage
diSPLaCE
HoMetoWn
pRohibit
FArAWAY
roAch
MAsk
0b58cti02
TAx
SatelLitE
9U7s9Q1E2tLY
vIo818
mICROPHoNe
ERR
pea
sw7At
elAbOrate
vanish
A638n87
obJECt
COnfUsed
MEaDow
never
HAPpeNinG
pencil
mOLecULE
explanatIon
WHY
side
pit
ChoCOLAtE
lingUIstics
MODERN275Ti04
eAger
NAIVE
tuNnel
DePArtuRE
InteRmedIaTe
emploY
coNTaMInAtE
CONStRUCtIon
CaTALOgue
AFFECT
LEND
expand
SLEEP
coUnseL
2602u21i0n
COndOm
RuNNEr
ASSaSsInAte
RADIATe
TOP
ASSISTANCe
BLOODY
8365M
powe8
peSSIMISM
PArT
kettle
sOLE
closE
536R1S8577
EMphASIs
MURDER
unusually
668H2P2
beg
s46E0
CONFUSE
TRAIN
DIEHARD
INNER
pERsuasIon
NEIgHbOr
MONSTROUS
trunk
KeTCHUp
COugH
KANGARoO
RIDE
deDiCate
FRagrANCe
sOb
SUpplY
PRinCE
SLIM
nONE
PAy
coStUMe
BRIefMg
