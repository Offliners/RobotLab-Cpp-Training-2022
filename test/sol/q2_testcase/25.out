Unknown
n. 發現
n. 柏拉圖(古希臘哲學家)
n. 耐力
n./v. 優雅
adj. 上等的
adv. 在將來某個時候
adj. 富裕的
n. 靈巧
adj. 廣闊的
adj. 提供忠告的
n. 觀光客
n. 傢俱
v. 觀察
n. 大屠殺
adj. 更適合的
v. 打算
n./v. 口水；吐
adj./adv./n./pron.  最後的；最近的
adj./adj. 緊的；緊緊地
n./v. 爭吵
n. 馬戲團
v. 持續
adj. 獨特的
n. 合唱隊
n. 說教；佈道
n. 海灣
n. 刀
n./v. 包；包裝
n. 後代
v. 使麻痺；使癱瘓
n. 角；號；喇叭
n. 行李；裝備
n. 聖歌
adj. 值得紀念的
n. 故事
n. 迷信
n. 觸鬚；天線
n. 道歉
n. 漁民；漁夫
pron. 誰
n./v. 衝突
n. 乳品店
v. 摒棄
adj. 有益的
adj. 有區別的
v. 吐露
Unknown
adj. 免除的
n. 可能的機會
n. 運動
n. 油；石油
adj. 總統(校長)的
n. 美國州議會大廈
adj./n. 十二
adv. 在起初
n. 方面；觀點
v. 使目眩
adj. 熱的；熱情的
adj. 手邊的；巫巧的
v. 信賴
n. 一套唱片；相簿
prep. 在…下面
Unknown
n. 性別
adj. 合作的
v. 鬼鬼祟祟地走
n. 斧頭
adj. 妒嫉的
adj. 不情願的；勉強的
v. 治癒
n. 降落傘
n. 宴會
n. 護士
n. 敵人
n. 高潮
n. 客人
Unknown
adj. 太陽的
v. 不信任
v. 同意；認可
Unknown
n./v. 地點；散布
n. 地主
n. 咖啡
n. 輪胎
n. (總稱)珠寶
v. 前往
n. 活動；活動力
n. 乞丐
n. 司法官
n. 創造力
adj. 肌肉的
n. 證書
adj. 不具名的
adj. 未料到地
v. 教導
n. 拖鞋
adj. 社會的
n. 保持
n. 可能性
adj./adv. 快速的
n. 愚蠢的錯誤
n. 教派
Unknown
n. 達到
v. 使驚駭
adj. 惡意的
adj./adv. 那時
n./v. 人群；擠滿
n./v. 回復；歸還
n. 口渴
n. 部隊
n. 網
n. 累積
n. 發薪日
adj. 想睡的
n. 立方體
v. 與…商量
n. 本文
pron. 誰
n. 手冊
n. 因素；代理人
n. 烏合之眾；無秩序的民眾
n./v. 呻吟
Unknown
n. 粉末
n./v. 目標；瞄準
n. 一分錢
adj. 保護的
n./v. 吐司；烤；烘
Unknown
n. 洩漏
n. 十字路口
v. 慫恿
Unknown
adj. 戰後的
adj. 貴重的；代價高的
adj. 一年一度的
n. 木材
n. 適應；改編(本)
adv. 不願意地
adj. 活著的；有活力的
n. 棒；沙洲；酒吧
Unknown
n. 勸告
adj./pron. (兩者之中)任一的
n. 甲板
n. 竹子
n./v. 震驚
n. 上下文
Unknown
n. 請願(書)
n. 護照
v. 蹲伏
v. 俯視
v. 擊退
n. 溼氣；水份
n. 跳蚤
n. 藥方
n./v. 聲音；表達
n. 改組
n./v. 車站；停止
adj. 可敬的
Unknown
n./v. 塔樓；屹立
n. 胡蘿蔔
n. 接受者
Unknown
n. 望遠鏡
n./v. 小河
n. 開業者
Unknown
v. 使縮短
Unknown
adj. 生物(學)的
n. 交響樂
n. 管子工
n. 國語
n./v. 榮譽；使…增光
v. 檢查；調查
n. 一套衣服
n. 爹地
n. 瀑布
n. 地板
v. 使害怕
n. 禮品；天賦
adj./pron. 另一個
Unknown
n. 出口；出路
adj. 平凡的
n. 科學家
n./v. 負擔；加負擔於…
n. 陸地；地球
v. 悲傷
aux. 應該
n. 電工；電氣技師
n. 健康
n. 電子學
n. 自助餐；快餐
n. 會計師
n. 復活節
v. 解釋；給…下定義
v. 浸；泡
n. 理解(力)
n. 隊
Unknown
Unknown
n. 直覺
n. 過敏性反應
n. 胸部
n. 限制
adj. 嚴密的
n./v. 下降
n. 同盟
Unknown
n./v. 製造業；製造
Unknown
v./n. 抓取；奪取
n. 驢；蠢人
v. 恐嚇
n./v. 鈕扣；按鈕
n. 豬
n. 一部分
n. 錄影帶
n./v. 等候
n. 蝴蝶
n./adj. 批發；大批的
n. 床
n. 宣布；聲明
v. 暴食
n. 諺語
n. 警告
adj. 殘忍的
v./n. 攻擊
n. 能力
adj. 實踐的
n. 馬拉松賽跑
n. 活力
ad v. 後來
n./v. 廚師；烹煮
n. 囚禁
adj. 可信的
v. 使驚訝
n. (美)元
adj. 小氣的
n. 基督教
n. 歷史學家
Unknown
v. 死亡
adv. 智力不足的；心理有缺陷的
adj. 溫和的；順口的
v. 查明
v.(n. )爭論；主張
n. 勾子；陷阱
Unknown
n. 揭露
n. 貨物
adv. 不知怎麼地
n./prop. 北京
n. 季節
n. 監獄
n. 鋼鐵
adj. 不法行為
adj. 愚蠢的
adj. 便利的
n. 法令；條例
v.(n. )鋪(用於被動)
n. 連接
adj. 冗長的
v. 堅持
n. 表(兄弟姊妹)
n. 製造者
n. 衛生
n. 救護車
adj. 重大的；嚴肅的
v. 同時共存
n. 抗體
adj. 聾的
Unknown
n. 卡片；紙牌遊戲
n. 工作日
n. 交通
n./v. 現金
Unknown
v. 發生
v. 邁大步走
n. 差別
n./v. 聲譽
n. 路邊
n. 花生
n. 建築師；設計師
adv. 基本上
n. 動機
adj. 重大的
adj. 感到抱歉的；感到遺憾的
n. 結構
adj. 歡樂的
adv. 在室內
n. 三明治
n. 沉思；冥想
n. 溝通；訊息
n. 提名；任命
Unknown
adv. 立即
Unknown
v. 爬行；蠕動
n./v. 要求；聲稱
n. 偽君子
n. 評論家
n. 典禮；禮儀
adj. 極好的
n. 庇護(所)
Unknown
n. 觀眾；聽眾
n./v. 尖叫聲；尖叫
n. 模型；模範
v. 歌唱
v. 殺死；使終止
n./v. 過程；加工
n./v. 戰役
Unknown
n. 紀念碑；紀念館
adj. 遮蔭的
adj. 嚇壞了的
n. 原野；欄位
n. 樣品
v. 駕駛
n./v. (繩等的)結
n. 獨處(狀況)
Unknown
n. 傳記
n. 成功
n. 精子
v. 違犯
adj./n. 巨人；巨大的
v. 解釋
n. 蜂蜜
n. 毛毯
n. 討厭的事物、人、行為等
n. 人行道
adj. 哲學的
Unknown
n. 燈籠；街燈
v. 孵蛋；策劃
n. 主廚
adj. 全球的
adj./v. 忙碌的
n. 生態學
n. 玉米；穀物
n. 珠寶
n. 數量
n./v. 轉動
n./v. 調查；審視
n. 表面
v. 競爭；妥善處理
v.(n. )發展
n. 意識形態
n./v. 借出
prep./n./adj. 加
n. 郡；縣
n. 蛋；卵細胞
adv. 以後
n. 皮夾子
Unknown
Unknown
n. 被提名人
adj./pron. 這些
n. 記號
n./v. 詭計；欺騙
n. 降雨
adj. 無恙的
n. 循環；流通；貨幣
n. 偽造
n. 民意調查
n. 流行性感冒
n. 相似
v. 使大吃一驚
n./v. 同意
adj. 愚蠢的
n. 羅曼蒂克
adj. 易於發生(不愉快事)的；有傾向的
n./v. 彎曲；扭曲
n. 煙霧
n. 派對
n./v. 鏡子；反映
n. 敵意
adj. 惡名昭彰的
Unknown
n./v. 冰鞋；溜冰
pron. 本身
n. 承認
n. 監護人；管理員
adj. 被動的
v. 使想起
v. 出現；顯露
v. 解答
n. 英畝數
n. 開始
adj. 不斷前進的
v. 監督
n. 角度；立場
adj. 繁榮的
n.  轉換
adj./v. 滿足的
Unknown
n. 警察
adj. 未觸動過的
n./v. 視力；看見
n./adj. 未來；前途
v./n. 摸；感覺
adj. 附加的
n./v. 滾動
adv. 向前
n./v. 地圖
n./v. 接觸；聯繫
n./v. 彎曲
adj. 逐步的；平緩的
n. 缺點
n. 外套大衣
n. 歡樂
n. 可信性
n. 創新；創造力
n./v. 旅行
n. 同意；認可
n./v. 圖畫；想像
adj. 小的
adj. 令人洩氣的
n. 慷慨
n. 長頸鹿
n. 颶風；暴風雨
n. 河流
v. 用力投擲
n./v. 表；把…編列成表
n./v. 損害；毀壞
v. 解釋；翻譯
adj. 家庭的；國內的
n. 計算機
n. 片刻
n. 危險
adj./n./v. 平均的
n. 身體；主要部分
n. 熱情
Unknown
n. 羊
n. 決心
n. 方法；態度
n. 多數
adj. 強壯的
n. 尾巴
n. 三月
adj./v. 正確的；更正
adj./v. 完全的；完成
n. 租約
n. 捐贈；捐贈物
n./v. 走
v. 移居於
n. 帳戶；描述
n. 丈夫
adv. 小心地
n. 家庭；家人
n. 代名詞
n. 公布
n. 口號
n. 服務員
n. 軍官；官員
adj. 陰險的
n. 小牛
n. 精製品
n. 澡缸；沐浴
adv./adj./n. 在城市的商業區；鬧區
n. 部門；系
n. 痛苦
n. 約定；指派
adv. 向下的
n./v. 救助
v. 埋伏突擊
v. 抑制
n. 道德敗壞
n. 女侍
v.  表達
adj. 奇妙的
v. 診斷
n. 領土
Unknown
n./v. 離婚
adj. 明顯的
n./v. 讚美；稱讚
v./n. 打擾；煩惱
n. 午餐；午宴
n. 牛肉
n. 選擇權；選擇力
n. 財產；好運
n. 分隔間
n. 外用藥水；化妝水
adj. 真誠的
Unknown
adj./n. 紫色的
n. 陽臺；樓座
adv. 反而
n. 兇手
n. 墓地
n. 服務員
n. 熊；魯莽之人
v. 專心
n./v. 打賭
prep. 不像
n. 爸爸(非正式)
v. 使適應
n. 成見
adj. 令人鼓舞的
n. 大陸；大洲
n. 暴行
v. 移開；取代
n. 永久居留地
v. 禁止
adj. 遠方的
n. 蟑鄉；大麻煙卷的煙蒂
n./v. 面具
Unknown
n./v. 稅；對…微稅
n. 衛星
Unknown
Unknown
n. 麥克風
v. 犯錯
n. 豌豆
Unknown
adj. 精巧的；複雜的
v. 消失
Unknown
n. 物體；目標
adj. 困惑的
n. 草地
adv. 永不
n. 事件
n. 鉛筆
n. 分子
n. 解釋
adv./pron./conj.  為什麼
n. 旁邊
n. 坑；煤礦
n. 巧克力
n. 語言學
Unknown
adj. 熱切的
adj.  天真的；樸素的
n. 隧道
n. 分離；啟程
adj. 中問的
v. (n)雇用
v. 污染物
n. 建造物；建設
n. 目錄
v. 影響；使感動
v. 把…借給
v. 展開；擴大
n./v. 睡眠
n. 忠告
Unknown
n. 保險套
n. 奔跑的人
v. 行刺
v. 發射
adj./nJv. 頂端的；頂部
n. 協助
adj. 血腥的
Unknown
Unknown
n. 悲觀(主義)
n. 一部分
n. 水壺
n. 腳底(板)；鞋底
adj./adv. 接近的
Unknown
n. 強調
n./v. 謀殺
adv. 非常
Unknown
v. 乞求；乞討
Unknown
v. 把…弄糊塗；攪亂
n. 火車
n. 頑固分子
adj. 內在的
n. 說服
n. 鄰居
adj. 怪物的；恐怖的
n. 樹幹
n. 調味蕃茄醬
v. 咳嗽
n. 袋鼠
n./v. 騎；騎乘
v. 奉獻
n. 芬芳
v. 啜泣
n./v. 供應
n. 王子
adj. 苗條的
pron./adv. 一點兒也沒
n/v. 付錢
n. 服裝；戲服
n. 簡報
